services/getting-hired-message.service.js

```diff
@@ -0,0 +1,52 @@
+const Redis = require('ioredis');
+
+class GettingHiredMessageService {
+  constructor() {
+    this.redis = new Redis(process.env.REDIS_URL);
+  }
+
+  async handleMessage(message) {
+    const userId = message.member.id;
+
+    try {
+      const userIsCached = await this.#isCached(userId);
+
+      if (!userIsCached) {
+        await this.redis.set(userId, true);
+        await GettingHiredMessageService.sendAuthorMessage(message);
+      } else {
+        // TODO: (REMOVE block) toggle the user's existence in the cache for testing purposes.
+        await this.redis.del(userId);
+      }
+    } catch (error) {
+      console.log('Error:', error);
+    }
+  }
+
+  static async sendAuthorMessage(message) {
+    /* TODO: Construct the message embed to send to the user, either via DM or in the channel
+    * If message cannot be DM'd it is sent in the channel. The message will need to mention or reply
+    * to the user.
+    */
+    const welcomeMessage = 'Test message';
```

Comment:
My suggestion: 
> "Welcome to the channel for the Getting Hired part of the curriculum. Please ensure you have completed the Getting Hired course and read all of the pins prior to engaging in this channel for resume review, interview help, or anything else covered in that section!"

Opting to not link to the getting hired module directly as the link will differ for Node/Ruby folks.